Documentation/network/standalone.rst

```diff
@@ -0,0 +1,1281 @@
+as a standalone gateway. The gateway is designed to reside at the edge of the
+cluster, potentially replacing dedicated hardware components, and handling
+high-traffic load from a north-south direction going into the cluster(s).
+
+The Cilium Standalone Gateway includes functionality such as a high performance
+eBPF-based layer 4 load-balancer and a NAT46/64 gateway. There is a wide range
+of use cases, for example, the Cilium Standalone Gateway could be used to
+frontend DNS resolvers, it can be used as a load-balancer for legacy VM-based
+workloads, as a gateway in front of IPv6 single stack Kubernetes clusters to
+allow for interconnection with IPv4, as a load-balancer for kubeapi-server, to
+replace legacy IPVS-based load-balancers, and many more.
+
+Cilium provides a programmable API for its agent which is exposed through a
+unix domain socket on the local node such that it can be integrated into
+third party management software on the node.
+
+This guide explains how to provision Cilium as a standalone gateway along with
+various use-cases and configuration examples.
+
+Quick-Start
+===========
+
+Given the Cilium Standalone Gateway can be operated outside of a Kubernetes
+cluster, this guide provides a minimal Docker-based step-by-step instructions.
+
+As a one-time prerequisite, ensure that the BPF file system is mounted under
+``/sys/fs/bpf``. If this is not the case already, a mount unit file can be set
+up under ``/etc/systemd/system/sys-fs-bpf.mount`` ...
+
+.. parsed-literal::
+
+   [Unit]
+   Description=BPF mounts
+   DefaultDependencies=no
+   Before=local-fs.target umount.target
+   After=swap.target
+
+   [Mount]
+   What=bpffs
+   Where=/sys/fs/bpf
+   Type=bpf
+
+   [Install]
+   WantedBy=multi-user.target
+
+... and brought up through ``systemctl start sys-fs-bpf.mount``.
+
+.. note::
+
+  The above step is only required for distributions with older systemd versions.
+  For most modern distributions this step is not required as systemd will
+  automatically mount the BPF file system at its default location. See also
+  `GH-10955 <https://github.com/cilium/cilium/issues/10955>`_ for details.
+
+In this Quick-Start example we enable the standalone layer 4 load-balancer
+functionality for both IPv4 and IPv6, as a load-balancing algorithm we
+utilize Maglev consistent hashing, and the networking device for the
+load-balancer in this case is called ``bond0``. Multi-device setups are
+supported as well, and can be specified via ``--devices=bond0,eth0`` or
+through a device name regex. Furthermore, we enable XDP acceleration for
+maximum efficiency.
+
+The vast majority of modern network drivers for 10/40/100G and beyond support
+XDP today. If you are unsure whether your network driver supports native XDP,
+then simply set ``--bpf-lb-acceleration=disabled`` instead and Cilium loads
+its programs into the tc BPF layer instead of XDP.
+
+.. parsed-literal::
+
+   docker run --name cilium-gateway -td \\
```

Comment:
Nit: s/cilium-gateway/cilium-loadbalancer/

Reply:
I see gateway term as a superset of both load-balancer and nat46/64 proxy.